libkbfs/interfaces.go

```diff
@@ -2012,6 +2012,10 @@ type BlockRetriever interface {
 	// Request retrieves blocks asynchronously.
 	Request(ctx context.Context, priority int, kmd KeyMetadata,
 		ptr BlockPointer, block Block, lifetime BlockCacheLifetime) <-chan error
+	// RequestNoPrefetch retrieves blocks asynchronously, but doesn't trigger a
```

Comment:
Does this mean it would get the block, but won't fetch its subtree?

Reply:
Yeah, basically it's a way for the prefetcher to use the cache just like everything else does, instead of caching its own blocks. If it used the regular `Request`, then every prefetch would trigger another prefetch for the block itself, recursively, forever.